apps/taxes/src/modules/avatax/order-created/avatax-order-created-mock-generator.ts

```diff
@@ -0,0 +1,70 @@
+import { TransactionModel } from "avatax/lib/models/TransactionModel";
+import { OrderCreatedSubscriptionFragment } from "../../../../generated/graphql";
+import { ChannelConfig } from "../../channels-configuration/channels-config";
+import { orderCreatedTransactionMock } from "./avatax-order-created-response-transaction-mock";
+import { AvataxConfig } from "../avatax-config";
+import { defaultOrder } from "../../../mocks";
+
+export type Order = OrderCreatedSubscriptionFragment;
```

Comment:
I dont like exporting `Order` from mock. Its causing spaghetti dependencies. 
Why don't use `OrderCreatedSubscriptionFragment` directly (its explicit that you reference gql payload), or `export OrderFragmentMock` ?